wordpress-mobile/WordPress-iOS
# Update uses of `accent` color with `primary`, to replace pink with blue

Fixes #15853. This PR replaces most uses of the `accent` color (pink) with `primary` (blue). I did consider redefining `accent` to use the blue color, but we still do need to use the pink in a couple of places like stats charts, and keeping a separate accent gives us more flexibility in the future.

As with [Android](https://github.com/wordpress-mobile/WordPress-Android/pull/14376), I've left a few elements using pink:

- Stats bar chart
- Small badges, such as the badge in the Help section of the app that shows the user has a new message, and the unread notifications badge in the tab bar

|   |   |   |   |
|---|---|---|---|
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 11 09](https://user-images.githubusercontent.com/4780/113303218-c5064800-92f8-11eb-8291-2c544620e9fc.png)  |  ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 12 03](https://user-images.githubusercontent.com/4780/113303242-cb94bf80-92f8-11eb-9620-ceab2c226a53.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 12 22](https://user-images.githubusercontent.com/4780/113303274-d18aa080-92f8-11eb-846f-223a595e87dd.png)  |  ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 17 00](https://user-images.githubusercontent.com/4780/113303322-df402600-92f8-11eb-902b-4d546e6ed970.png) |
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 34 08](https://user-images.githubusercontent.com/4780/113303345-e5360700-92f8-11eb-926d-2db39067561e.png)  |  ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 16 41](https://user-images.githubusercontent.com/4780/113303376-ed8e4200-92f8-11eb-9876-f5ef2d9ce90d.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-03-31 at 20 46 45](https://user-images.githubusercontent.com/4780/113303423-faab3100-92f8-11eb-8ca5-874e50582511.png)  |  ![Simulator Screen Shot - iPhone 12 Pro - 2021-03-31 at 20 45 21](https://user-images.githubusercontent.com/4780/113303443-00a11200-92f9-11eb-8268-366ea889a016.png) |
|  ![Simulator Screen Shot - iPhone 12 Pro - 2021-03-31 at 20 44 46](https://user-images.githubusercontent.com/4780/113303463-0565c600-92f9-11eb-8307-64ff8fd257e3.png) |  ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 49 06](https://user-images.githubusercontent.com/4780/113303917-7311f200-92f9-11eb-81c7-dab216608bb7.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 48 50](https://user-images.githubusercontent.com/4780/113303924-74431f00-92f9-11eb-8bc5-ff9327494429.png)  |  ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 47 41](https://user-images.githubusercontent.com/4780/113303934-75744c00-92f9-11eb-86a7-f18bce521d70.png) |
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 14 47 28](https://user-images.githubusercontent.com/4780/113303935-760ce280-92f9-11eb-899e-7ab52bcc6119.png)  |   |   |   |


**To test**

- Build and run. Test in light and dark mode.
- Go through the main flows - Login, Create a site, Quick start
- Check the buttons and the links are all blue now
- Check nothing is out of place
- Be cautious about dialogs

**Also**

-  Go to stats with a lot of data
- Go to Year view
- Notice the selected day in the bar chart is still pink
- Scroll to the Countries block
- Notice the countries are now in blue

## Regression Notes
1. Potential unintended areas of impact

The entire app

2. What I did to test those areas of impact (or what existing automated tests I relied on)

I went through the login/create a site flow, smoke tested stats, notifications, comments and other areas of the app

3. What automated tests I added (or what prevented me from doing so)

None

PR submission checklist:


cc @leandroalonso @momo-ozawa Would one of you mind doing a quick check through the Jetpack build and ensure I haven't messed anything up there?
